model/installation_request.go

```diff
@@ -22,18 +22,19 @@ import (
 
 // CreateInstallationRequest specifies the parameters for a new installation.
 type CreateInstallationRequest struct {
-	OwnerID         string
-	GroupID         string
-	Version         string
-	Image           string
-	DNS             string
-	License         string
-	Size            string
-	Affinity        string
-	Database        string
-	Filestore       string
-	APISecurityLock bool
-	MattermostEnv   EnvVarMap
+	OwnerID          string
+	GroupID          string
+	Version          string
+	Image            string
+	DNS              string
+	License          string
+	Size             string
+	Affinity         string
+	Database         string
+	Filestore        string
+	APISecurityLock  bool
+	MattermostEnv    EnvVarMap
+	ExtraAnnotations []string
```

Comment:
My reasoning for the name `ExtraAnnotations` was that we will probably be adding some annotations automatically in the future like for example `multi-tenant` or `private` based on the Installations affinity. That is why I wanted to highlight that those passed to the API are extra ones.

I am open to change it, just wanted to give you full context, let me know what do you think.

Reply:
Hmm, can you clarify how the provisioner would know which annotations should be automatically added? Affinity here would simply be a result of which annotations we provided when creating the installation.

Reply:
Sure, after some discussion in a design doc, we concluded that for Installations it would be good to add the annotation based on the affinity. For example, the multi-tenant installation would automatically get `multi-tenant` annotation, to prevent having installations without any annotations, as such Installation could be scheduled on any Cluster.

Reply:
As discussed, changed the `ExtraAnnotations` to `Annotations`.